models/common/model.go

```diff
@@ -3,8 +3,8 @@ package common
 
 import "time"
 
-// Model is a common model that contains common fields for all models.
-type Model struct {
+// OldModel is a common model that contains common fields for all models.
```

Comment:
If we stay with comment let's add why it's named "Old" - few words about deprecation will be enough.

Reply:
added Deprecation message